src/features/Pagenation/Button/index.module.css

```
@@ -34,7 +34,7 @@
 }
 
 .page-btn--active {
-  background: #f6b000; /* 黄色 */
```

この黄色はできればパレットに登録しているプライマリーカラーを使って反映して欲しい。
https://github.com/simesaba80/toybox-front/blob/47dd1944a5039db499f664d7c2b7f4753428e52b/src/index.css#L7


```suggestion
  background: var(--primary-color);
```